relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -719,6 +726,16 @@ fn sql_table_from_query(query: &str) -> Option<&str> {
     extract_captured_substring(query, &SQL_TABLE_EXTRACTOR_REGEX)
 }
 
+/// Regex with a capture group to extract the HTTP method from a string.
+static HTTP_METHOD_EXTRACTOR_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(r#"(?i)^(?P<method>(GET|HEAD|POST|PUT|DELETE|CONNECT|OPTIONS|TRACE|PATCH))\s"#)
```

Comment:
I've taken the list of these methods from [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).